plugins/colorschemes/catppuccin.nix

```diff
@@ -115,23 +115,30 @@ in {
           "Define operators highlight properties";
       };
 
-      colorOverrides = helpers.defaultNullOpts.mkNullable (types.attrsOf types.str) "{}" ''
-        Define color overrides
-        Example:
-         color_overrides = {
-             all = {
-                 text = "#ffffff",
-             },
-             latte = {
-                 base = "#ff0000",
-                 mantle = "#242424",
-                 crust = "#474747",
-             },
-             frappe = {},
-             macchiato = {},
-             mocha = {},
-         }
-      '';
+      colorOverrides =
+        helpers.defaultNullOpts.mkNullable
+        (with types; attrsOf (attrsOf str))
```

Comment:
the keys can only be the same values than the `flavour` maybe a submodule instead of an attrsOf for the first one ?

Reply:
Done ! Is it OK for you ?